bundles/org.openhab.binding.boschindego/README.md

```diff
@@ -126,6 +126,13 @@ Number:Area              Indego_GardenSize         { channel="boschindego:indego
 Image                    Indego_GardenMap          { channel="boschindego:indego:singlekey:lawnmower:gardenMap" }
 ```
 
+---
+**NOTE**
+
+Currently some models (verified with Indego S+500 and Indego M+700) the channel `batteryVoltage` is not properly populated with the actual battery voltage but with the `batteryLevel`.
```

Comment:
A few minor suggestions:
- Skip mentioning specific models. It seems unneeded and can become misleading if firmware or service will be fixed.
- Slightly rephrase to avoid any confusion between a problem in the models vs. a problem with the binding's channels.
- Directly advise which channel to use.

```suggestion
Currently some models report the battery level in percent as voltage.
For these models `batteryLevel` should be preferred over `batteryVoltage`.
```

Reply:
Sounds good - adapted the text.